ext/mysql2/client.c

```diff
@@ -1464,4 +1501,17 @@ void init_mysql2_client() {
   rb_const_set(cMysql2Client, rb_intern("BASIC_FLAGS"),
       LONG2NUM(CLIENT_BASIC_FLAGS));
 #endif
+#ifdef SSL_MODE_SUPPORT
```

Comment:
SSL_MODE_DISABLED, etc are not #define'd in mysql.h, they're enums... hence the ugly hack with the version stuff in extconf.rb